### DISCUSSION TITLE
Projects Pipelines: Link Pull Requests to Issues

### DISCUSSION THREAD

**Discussion by cnewtonMYCM** [community/community#86721] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Projects

### Body

Hello, this is a feature request for Pipelines in GitHub Projects.

It'd be great if we could link our pull requests to issues.

Right now, we're manually putting our pull requests below the issues like so:
![1](https://github.com/community/community/assets/119305634/0a972523-02aa-466c-a678-007dc49eaa7c)

But it'd be nice if they could be linked so that if you move one, the other automatically moves with it.

FYI: With Zenhub, you can link your Pull Requests to Issues like so:
![2](https://github.com/community/community/assets/119305634/cace123a-a5a5-4208-b6c7-665d1f59a60f)

Thank you.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by cparkinsonMYCM** [comment]
To add a little context, having the project display the linked pull requests is fine to a point:
![image](https://github.com/community/community/assets/47317809/3f2e036c-d562-43f9-9e27-d52c6df4b934)

But when PR 2065 in this image (from one repository) is merged into the main branch, issue #992 will be closed and moved to the `Done` column, but PR 62 (from a different repository, same issue present in web app and mobile app...) will not have been dealt with yet.

The visual indicator here is not enough to easily notice from the closed issue that PR 62 needs attention. We are currently managing this separately.
And if we include the pull requests in the projects, then the management of their position in the columns becomes more of a chore and less productive...

But if the cards were visibly linked/nested that would become more clearer and easier to manage.

Also relevant, this problem is compounded with the problem the following discussion is addressing:
https://github.com/orgs/community/discussions/17308
